libmate-desktop/mate-desktop.h.in

```diff
@@ -26,20 +26,6 @@
 #ifndef __MATE_DESKTOP_H__
 #define __MATE_DESKTOP_H__
 
-#ifdef HAVE_CONFIG_H
-#include <config.h>
-#endif
-
-#include <glib.h>
-
-G_BEGIN_DECLS
-
-#define MATE_DESKTOP_CHECK_VERSION(major,minor,micro) \
-        (MATE_MAJOR > (major) || \
-        (MATE_MAJOR == (major) && MATE_MINOR > (minor)) || \
-        (MATE_MAJOR == (major) && MATE_MINOR == (minor) && \
-        MATE_MICRO >= (micro)))
```

Comment:
We can keep that if we want, just replace `MATE_MAJOR` & al. to it's actual value.

Reply:
```MATE_DESKTOP_VERSION``` is sufficient to solve all the problem

Reply:
As you want, I just pushed a working version of `MATE_DESKTOP_CHECK_VERSION`, but feel free to drop it if it's useless, I don't mind.

Reply:
Thank you